customer_team_manager/static/src/js/customer_admin_navbar.esm.js

```diff
@@ -0,0 +1,23 @@
+/** @odoo-module */
+
+import {NavBar} from "@web/webclient/navbar/navbar";
+import {patch} from "@web/core/utils/patch";
+import {session} from "@web/session";
+
+patch(NavBar.prototype, "customer_team_manager_navbar", {
+    /**
+     * @override
+     * Filters all mail-related NavBar menu items from the Systray Items,
+     * if the current user is a customer admin (has the `group_customer_admin` group)
+     *
+     * We do this since these menus try to access records only accessible to internal users,
+     * which can provoke an access error to customer admins (portal users).
+     */
+    get systrayItems() {
+        var navbar_items = this._super();
+        if (session.is_customer_admin) {
```

Comment:
On ne peut pas plutôt transformer le test en "n'est pas un utilisateur interne" ? (pas sûr qu'on ait l'info). Ce serait plus logique et plus "future proof".

En passant, typo dans le message de commit : "accessible for" => "accessible by".